### ISSUE TITLE
A11y - User create page

### ISSUE THREAD

**Issue by luistoptal** [gigascience/gigadb-website#1387]
## User story

Prio 3:

- [ ] Fix heading hyerarchy
- [ ] Fix contrast issues
- [ ] Fix landmark usage (empty or contentless sections)
- [x] Make input focus state more prominent
- [x] Make checkbox focus state visible
- [x] make form errors a11y (https://webaim.org/techniques/formvalidation/#error)
- [x] required labels should be indicated to screen readers (aria-required="true", required attr)
- [x] Associate newsletter checkbox to its label and description
- [x] `<font>` element is deprecated, replace it by something proper, ref https://developer.mozilla.org/en-US/docs/Web/HTML/Element/font
- [ ] Make Captcha more accessible (probably create separate ticket for accessible captchas that can be reused in different forms), read: https://www.w3.org/TR/turingtest/#:~:text=CAPTCHAs%20are%2C%20consequently%2C%20allowed%20under,sensory%20perception%20are%20provided%20to

Prio 2

- [x] Consider adding client side validation that triggers on submission attempt and/or on input blur (ref: https://webaim.org/techniques/formvalidation/)

https://beta.gigadb.org/user/create

**Comment by luistoptal** [comment]
https://github.com/gigascience/gigadb-website/pull/1439 will fix some of these issues:

* input focus state

It's probably a good idea to wait for that PR to be reviewed before working on this one
